### DISCUSSION TITLE
Feature Request: Ability to create a unique instance of a symbol?

### DISCUSSION THREAD

**Discussion by CBrosM** [keiwando/toonsquid#324] [Ideas]
Once a symbol layer is created, have the ability to copy and paste a unique instance of it versus duplicating where every nested clip is linked.

**Comment by Cat-Butler** [comment]
I suspect you almost certainly know this already: 
As an interim workaround until this feature is included, duplicate the symbol in the Library to create another symbol that you can give unique properties, or use as a separate unique instance. 
Tap-**and-hold** on the symbol in the Library, select Duplicate from the choices that appear along the bottom edge of the Library window.
![dup](https://github.com/keiwando/toonsquid/assets/25494772/5c4196c8-794c-4d61-8755-6b8f066a92dc)

**Reply by CBrosM** [reply]
Thank you so much for replying - I did not know this. It’s a perfect workaround for what I was trying to achieve.
